EIPS/eip-4750.md

```diff
@@ -116,17 +116,19 @@ In addition to container format validation rules above, we extend code section v
     1. Code section is invalid in case offset points to a position outside of section bounds.
     2. Code section is invalid in case offset points to one of two bytes directly following `CALLF` instruction.
 
+### Deprecated instructions
+
+Dynamic jump instructions `JUMP` (`0x56`) and `JUMPI` (`0x57`) are depreceated and invalid and their opcodes do not have any instruction assigned.
```

Comment:
```suggestion
Dynamic jump instructions `JUMP` (`0x56`) and `JUMPI` (`0x57`) are depreceated and invalid and their opcodes are undefined.
```

Not sure this is better, but I like "undefined opcode" and/or "undefined instruction" term.

Reply:
Undefined sounds fine.